### PR TITLE
Azure release 9.0.0 with Kubernetes 1.15.5

### PR DESCRIPTION
**Pull request by tfussell**
Towards https://github.com/giantswarm/giantswarm/issues/6431

Starting with this release, release notes are included as a permanent record in markdown form in this repo. The Slack formatted release notes are here https://gigantic.slack.com/archives/CHAHY6VMF/p1572019708004300

Split off from https://github.com/giantswarm/releases/pull/17 in the interest of speed.

### CODE REVIEW

**Comment by tfussell** [comment]
You mean create `v12` in `azure-operator` @MarcelMue?

**Comment by MarcelMue** [comment]
> You mean create `v12` in `azure-operator` @MarcelMue?

Yes, so we have incremented versions in the new WIP

File: azure.yaml [hunk 1]
```diff
@@ -1,4 +1,24 @@
...
+  - endpoint: http://cert-operator:8000
+    name: cert-operator
+    version: 0.1.0
+  - endpoint: http://chart-operator:8000
+    name: chart-operator
+    version: 0.7.0
+  - endpoint: http://cluster-operator:8000
+    name: cluster-operator
+    provider: azure
+    version: 0.21.0
```
Comment (rossf7) [inline azure.yaml]: @tfussell @MarcelMue 

cluster-operator 0.22.0 was wired in https://github.com/giantswarm/cluster-operator/pull/752 

Can we use it here?

Comment (MarcelMue) [reply azure.yaml]: Yes, we should update this to 0.22.0

**Pull request merged**